src/solvers/refinement/string_constraint_generator.h

```diff
@@ -363,18 +370,21 @@ class string_constraint_generatort: messaget
     exprt char1, exprt char2, exprt char_a, exprt char_A, exprt char_Z);
   bool is_constant_string(const string_exprt &expr) const;
   string_exprt empty_string(const refined_string_typet &ref_type);
+  unsigned long to_integer_or_default(const exprt &expr, unsigned long def);
```

Comment:
It's fine I guess. I think in other places we stuck to `mp_integer` for consistency, but whatever you feel is better.